web/app/js/components/TapQueryForm.jsx

```diff
@@ -91,11 +89,13 @@ class TapQueryForm extends React.Component {
   static propTypes = {
     classes: PropTypes.shape({}).isRequired,
     cmdName: PropTypes.string.isRequired,
+    currentQuery: tapQueryPropType.isRequired,
```

Comment:
This `prop` name has been changed due to `withQueryParams ` HOC injects the props `query` and `setQuery` into the component based on the config.

Reply:
Thanks for the explanation!